src/index.html

```diff
@@ -3,11 +3,52 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="laptop"
+      >
+
+      <div class="card__info">
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+      </div>
+
+      <div class="card__price">
+        <div>Price:</div>
+        <div class="value">$2,199</div>
+      </div>
+      <button  class="card__btn" data-qa="hover">
```

Comment:
```suggestion
      <button class="card__btn" data-qa="hover">
```

remove please redundant space